content_fetch/fetch_gc_articles_blog_post.js

```diff
@@ -2,56 +2,77 @@ const fetch = require("node-fetch");
 const buildFileName = require("../utils/buildFileName");
 
 var getBlogPostsFromGCArticles = async function (lang) {
-  let url =
+  let page = 1;
+  const perPage = 100;
+  let baseUrl =
     lang == "en"
-      ? process.env.GC_ARTICLES_ENDPOINT_EN + "posts?markdown=true&_embed"
-      : process.env.GC_ARTICLES_ENDPOINT_FR + "posts?markdown=true&_embed";
-  return await fetch(url)
-    .then((response) => response.json())
-    .then((data) => {
-      let files = [];
-      for (p in data) {
-        let post = data[p];
-        let replacedTitle = post.title.rendered.replace(/&#8217;/g, "’");
-        let category = post._embedded["wp:term"];
-        let categoryNames = [];
-        let out = "";
-        out += "---\n";
-        out += "layout: blog\n";
-        out += "title: '" + replacedTitle + "'\n";
-        out += "description: >-\n";
-        out += "  " + post.markdown.excerpt.rendered + "\n";
-        out += "author: '" + post.meta.gc_author_name + "'\n";
-        out += "date: '" + post.date + "'\n";
-        if ( post._embedded[ 'wp:featuredmedia' ] ){
-          out += `image: ${ post._embedded[ 'wp:featuredmedia' ][ 0 ].media_details.sizes.full.source_url }\n`;
-          out += `image-alt: ${ post._embedded[ 'wp:featuredmedia' ][ 0 ].alt_text }\n`;
-          out += `thumb: ${ post._embedded[ 'wp:featuredmedia' ][ 0 ].media_details.sizes.full.source_url }\n`;
-        }
-        if (category) {
-          category.forEach((group) => {
-            group.forEach((term) => {
-              categoryNames.push(term.name);
-            });
-          });
-        } else {
-          categoryNames = [""];
-        }
-        // out += "tags: " + JSON.stringify(categoryNames) + "\n";
-        out += "translationKey: " + post.slug + "\n";
-        out += "---\n";
+      ? process.env.GC_ARTICLES_ENDPOINT_EN
+      : process.env.GC_ARTICLES_ENDPOINT_FR;
+
 
-        // Convert any body image URLS to Cloudfront
+  let files = [];
+  
+  while (true) {
+    const url = `${baseUrl}?page=${page}&per_page=${perPage}`;
```

Comment:
Do we still need this original params on the url?
```suggestion
    const url = `${baseUrl}posts?markdown=true&_embed&page=${page}&per_page=${perPage}`;
```